docs/LearningOutcomes.adoc

```diff
@@ -37,7 +37,7 @@ Note how the <<DeveloperGuide#architecture, Developer Guide>> uses events to com
 
 == Use API Design `[LO-ApiDesign]`
 
-Note how components of AddressBook have well-defined APIs. For example, the API of the `Logic` component is given in the link:{repoURL}/src/main/java/seedu/address/logic/Logic.java[`Logic.java`]
+Note how components of ProgressChecker have well-defined APIs. For example, the API of the `Logic` component is given in the link:{repoURL}/src/main/java/seedu/address/logic/Logic.java[`Logic.java`]
```

Comment:
yes you are right

Reply:
actually the other way round. I realized the LearningOutcome file is refering to the original addressbook-level4 instead of our current project. moreover, in the end the LO file will be removed in our final product